platform-hub-api/config/routes.rb

```diff
@@ -33,7 +33,7 @@
     end
 
     resources :users, only: [ :index, :show ] do
-      get '/search/:q', to: 'users#search', on: :collection
```

Comment:
Nice! Would you mind adding a spec case for this in `users_routing_spec.rb`?

Reply:
Sure, added 👍